boards/nrf52840dongle/Makefile.features

```diff
@@ -4,6 +4,7 @@ CPU_MODEL = nrf52840xxaa
 FEATURES_PROVIDED += periph_pwm
 FEATURES_PROVIDED += periph_uart
 FEATURES_PROVIDED += periph_usbdev
+FEATURES_PROVIDED += periph_spi
```

Comment:
The board's Kconfig will need an update too

Reply:
Ow, I did not know that that existed.
I do not know how to test the Kconfig, but I think it is added now.

Reply:
It's tested by `tests/kconfig_features` - but this is a trivial change to begin with.

Please squash!